### ISSUE TITLE
Sliver backdoor in one of the dependencies

### ISSUE THREAD

**Issue by farfiree** [sensepost/gowitness#238]
I think there is a (Sliver) backdoor in one of the dependencies. I'm getting an error while running:
`go install github.com/sensepost/gowitness@latest`

```
Type:           Backdoor
Name:           OSX/Sliver.ext
Infected file:  /private/var/folders/_w/44dlnjtx0nqg9k3g2ygx627c0000gn/T/go-build987384094/b001/exe/a.out
```

What's up with that

**Comment by leonjza** [comment]
I doubt that is a true positive. What generated that output? Are you sure it's related to gowitness and not a sliver install you have?

I uploaded the compiled ARM darwin artefact to VirusTotal out of interest anyways. https://www.virustotal.com/gui/file/713f81ab9889e03dd07f5a56a9a4bed3d51ce133bbaef6f612dfee1b5effa529?nocache=1

**Comment by leonjza** [comment]
Also, can you share that `a.out` file and hash, please?

**Comment by farfiree** [comment]
Also to add: I do not have Sliver framework installed. I'm familiar with it but it's not installed. Running the go install command on a different mac gives the same result.

**Comment by leonjza** [comment]
Unfortunately I don't know Intego AV, (it also doesn't seem to be listed on VirusTotal). I'm going to go with a false positive in your AV for now given the 64 vendors being okay with the arm Darwin binary at least.

Feel free to build and upload your own (in a sandbox without your AV) to test, ofc!

If you learn anything more about why this is happening, feel free to add, thanks!

**Comment by leonjza** [comment]
Sorry and just to be clear, there should be no sliver related dependency / artefact in gowitness.

**Comment by leonjza** [comment]
I think it's related to this issue: https://github.com/ysmood/leakless/issues/8, which means I may temporarily remove the go-rod driver until that is resolved.

**Comment by cardassian-tailor** [comment]
Hi - are there any updates on this?

**Comment by leonjza** [comment]
Best to refer to the issue linked in the previous post for the latest on it.

**Comment by cardassian-tailor** [comment]
> Best to refer to the issue linked in the previous post for the latest on it.

Thank you, is it safe to consider the tool (gowitness) unsafe to use until resolved?

**Comment by leonjza** [comment]
> > Best to refer to the issue linked in the previous post for the latest on it.
> 
> Thank you, is it safe to consider the tool (gowitness) unsafe to use until resolved?

No, but you can read the source of leakless to make up your mind ;)

**Comment by singe** [comment]
Leon did you check leakless’ source? Are you comfortable using it?

**Comment by leonjza** [comment]
Yeah - as an alternative I can update this repo to implement it manually using `go generate` and  a `replace` directive though I doubt it would prevent AV's from not liking the execution behaviour leakless uses. 

Alternatively we can remove go-rod support, but I'm not convinced there is anything nefarious happening here yet. Happy to be proven otherwise.

**Comment by singe** [comment]
That’s good enough for me. Thank you.
